android/app/build.gradle

```diff
@@ -105,8 +115,11 @@ android {
         applicationId "com.martinogg.coffeepro"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 3
```

Comment:
get back to versioncode 4 / 1.1.0 because next version of app will need to update from here.